Packet++/header/SomeIpTpLayer.h

```diff
@@ -0,0 +1,134 @@
+#ifndef PACKETPP_SOMEIPTP_LAYER

```

Comment:
We usually try to keep all classes of a layer in the same file. I'd move `SomeIpTpLayer` to `SomeIpLayer.h`, and the implementation to `SomeIpLayer.cpp`